menas/ui/components/dataset/conformanceRule/SchemaManager.js

```diff
@@ -16,11 +16,13 @@
 class SchemaManager {
 
   static getTransitiveSchemas(schemas, rules) {
-    rules.map(RuleFactory.createRule).forEach((rule, index) => {
-      const schema = $.extend(true, [], schemas[index]);
-      rule.apply(schema.fields);
-      schemas.push(schema);
-    })
+    if(rules) {
```

Comment:
I would use the return early pattern. This is JS and JS doesn't mind using `if (!rules) { return }` and then next code